orange/src/main/java/com/hartwig/hmftools/orange/algo/purple/PurpleInterpreter.java

```diff
@@ -444,4 +452,15 @@ private static PurpleCharacteristics createCharacteristics(@NotNull PurpleData p
                 .svTumorMutationalBurden(purple.purityContext().svTumorMutationalBurden())
                 .build();
     }
+
+    @NotNull
+    private static ChromosomalRearrangements createSuspectChromosomalRearrangments(@NotNull PurpleData purple,
```

Comment:
Why are these arrangements "suspect"? 

It may actually be suspect but that doesn't reflect anywhere else in the code. I would either go for suspect-all-the-way or not-at-all.